[WFLY-21138][WFLY-21137]: Feature-spec generation treats all host parameters as bound to domain mode and Galleon upgrade

ehsavoie opened the pull request:
* WFCORE-7372: Renaming host and profile address parameters to __host and __profile  when they are the 1st elements of the address to avoid confusion
* WFLY-21137: Updating Galleon and Galleon plugins to align with Core   

Requires https://github.com/wildfly/wildfly-core/pull/6532 to be merged and core to be updated.
  
Jira: 
 - https://issues.redhat.com/browse/WFLY-21138
 - https://issues.redhat.com/browse/WFLY-21137 (Galleon upgrade required)

ehsavoie committed:
[WFLY-21137]: Upgrade Galleon to 6.1.1.Final and Galleon plugins to 8.0.2.Final.

* Aligning versions with WilFly Core

Jira: https://issues.redhat.com/browse/WFLY-21137

Signed-off-by: Emmanuel Hugonnet <ehugonne@redhat.com>

yersan reviewed (commented):
Hello @ehsavoie , this is a WildFly change, so it should be tracked on its own WFLY Jira

I'll kick off integration Jobs, but paperwork should be done accordingly

ehsavoie commented:
@yersan everything should be ok now

yersan reviewed (approved):
Integration tests are green, so this looks good to me and will be incorporated into the next WildFly Core Release

ehsavoie committed:
[WFLY-21138]: Feature-spec generation treats all host parameters as bound to domain mode.

* host and profile address parameters are renamed to __host and __profile to avoid confusion

Jira: https://issues.redhat.com/browse/WFLY-21138
Core Issue: https://issues.redhat.com/browse/WFCORE-7372

Signed-off-by: Emmanuel Hugonnet <ehugonne@redhat.com>
